Userland/DevTools/Playground/main.cpp

```diff
@@ -104,6 +104,35 @@ int main(int argc, char** argv)
     editor.set_should_autocomplete_automatically(true);
     editor.set_automatic_indentation_enabled(true);
 
+    RefPtr<GUI::Action> m_save_as_action = GUI::CommonActions::make_save_as_action([&](auto&) {
```

Comment:
The `m_` prefix is only for member variables. Just use the same pattern as with the quit action instead (no temporary variable)